test/OpenTelemetry.Instrumentation.GrpcClient.Tests/GrpcTagHelperTests.cs

```diff
@@ -53,7 +53,7 @@ public void GrpcTagHelper_TryParseRpcServiceAndRpcMethod(string grpcMethod, bool
         public void GrpcTagHelper_GetGrpcStatusCodeFromActivity()
         {
             var activity = new Activity("operationName");
-            activity.AddTag(GrpcTagHelper.GrpcStatusCodeTagName, "0");
+            activity.SetTag(GrpcTagHelper.GrpcStatusCodeTagName, "0");
```

Comment:
Not sure if "0" should be int or string. But dropping a comment to point it out.